documentation/configuration.md

```diff
@@ -210,6 +210,10 @@ Required settings:
 * `DEFAULT_INSTANCE_RABBITMQ_URL`: The RabbitMQ AMQPS URI to be used by instances. E.g.,
   `amqps://rabbitmq.example.com:5671`
 
+### Redis settings
+* `DEFAULT_INSTANCE_REDIS_URL` The Redis URI (including the protocol, port, db, and basic auth)
+  to be used by instances. E.g., `rediss://admin:admin_password@redis.example.com:6397/0`
```

Comment:
@gabor-boros can we add a comment that this is processed into the database with `RedisServerManager._create_default` - so we can see the link between this and the `RedisServer` model?

Reply:
Fixed in 53688046